src/position.ts

```diff
@@ -663,6 +663,20 @@ export class Position {
 	}
 
 
+	/**
+	 * Whether both players have insufficient material so the game cannot end in checkmate. If the position is not legal (see {@link Position.isLegal}),
```

Comment:
The check on `isLegal(..)` is missing in the method implementation.

Reply:
done.